be/tests/Feature/HTTP/AuthenticationTest.php

```diff
@@ -0,0 +1,91 @@
+<?php
+
+namespace Tests\Feature\HTTP;
+
+use App\Models\User;
+use Illuminate\Foundation\Testing\RefreshDatabase;
+use Illuminate\Foundation\Testing\WithFaker;
+use Tests\TestCase;
+
+class AuthenticationTest extends TestCase
+{
+    public function test_registration_api_with_processable_content()
+    {
+        $res = $this->withHeaders([
+            'Accept' => 'application/json',
+            'Content-type' => 'application/json'
+        ])->json(
+            'POST',
+            '/api/register',
+            [
+                "first_name" => fake()->firstName(),
+                "middle_name" => fake()->firstName(),
+                "last_name" => fake()->lastName(),
+                "email" => fake()->unique()->safeEmail(),
+                "password_confirmation" => 'password',
+                "password" => 'password',
+            ]
+        );
+        $res->assertStatus(201);
+    }
+
+    public function test_registration_api_with_unprocessable_content_return_validation_message()
+    {
+        $res = $this->withHeaders([
+            'Accept' => 'application/json',
+            'Content-type' => 'application/json',
+        ])->json(
+            'POST',
+            '/api/register',
+        );
+        $res->assertStatus(422);
```

Comment:
- this is not model testing. this is controller testing. You should only test the functions you created in each of your models